### DISCUSSION TITLE
[Feature Request]: How to get the relaxed structure

### DISCUSSION THREAD

**Discussion by Yong-Q** [CederGroupHub/chgnet#90] [Announcements]
### Email (Optional)

3244608729@qq.com

### Problem

**structure.perturb(0.1)
result = relaxer.relax(structure, verbose=False)
print(f"\nCHGNet took {len(result['trajectory'])} steps. Relaxed structure:")
print(result["final_structure"])**
How to get the relaxed structure, the error of 30mev/atom in the article, is it relaxed?

### Proposed Solution

Hope to get the final structure of cif.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bowen-bd** [comment]
Please read[ pymatgen structure doc](https://pymatgen.org/pymatgen.core.html)

**Reply by kaijiz** [reply]
Hi, i also met this problem, I don't know how to output the relaxed structure into cif or POSCAR format. I searched pymatgen doc and didn't find. Could you tell me how to do?

**Reply by janosh** [reply]
```py
relaxer.relax(structure)["final_structure"].to(fmt="cif")
```

**Reply by kaijiz** [reply]
Thanks so much for replying to me so late!
